lib/auth/join.go

```diff
@@ -162,7 +166,8 @@ func (a *Server) generateCerts(ctx context.Context, provisionToken types.Provisi
 			renewable = true
 		case types.JoinMethodIAM,
 			types.JoinMethodGitHub,
-			types.JoinMethodCircleCI:
+			types.JoinMethodCircleCI,
+			types.JoinMethodKubernetes:
```

Comment:
oh you will also need an update here for this

https://github.com/gravitational/teleport/blob/51e8b3677b6125ea82b245d7dffff233e12f326a/lib/tbot/renew.go#L501-L507

Reply:
I disabled tbot support until either tbot or the operator supports running in kube and I have a tested setup.